packages/scandipwa/src/component/NavigationTabs/NavigationTabs.container.js

```diff
@@ -136,6 +146,9 @@ export class NavigationTabsContainer extends NavigationAbstractContainer {
         const offset = window.innerHeight + window.pageYOffset;
         const height = doc.scrollHeight;
 
+        const { isNotificationVisable } = this.props;
+        const { isScrolling } = this.state;
```

Comment:
I believe the `isScrolling` is a misnomer here.
You don't check whether the page is scrolling here.
You just set the boolean flag here and exit the function if the flag is true.
Please put the proper name.